File: src/test/java/leets/leenk/domain/feed/test/LinkedUserTestFixture.java
package leets.leenk.domain.feed.test;

import leets.leenk.domain.feed.domain.entity.Feed;
import leets.leenk.domain.feed.domain.entity.LinkedUser;
import leets.leenk.domain.user.domain.entity.User;

public class LinkedUserTestFixture {
    public static LinkedUser createLinkedUser(Long id, User user, Feed feed) {
        return LinkedUser.builder()
                .id(id)
                .feed(feed)
                .user(user)
                .build();
    }
}

File: src/test/kotlin/leets/leenk/domain/feed/test/LinkedUserTestFixture.kt
package leets.leenk.domain.feed.test

import leets.leenk.domain.feed.domain.entity.Feed
import leets.leenk.domain.feed.domain.entity.LinkedUser
import leets.leenk.domain.user.domain.entity.User

object LinkedUserTestFixture {
    fun createLinkedUser(
        id: Long?,
        user: User,
        feed: Feed,
    ): LinkedUser =
        LinkedUser(
            id = id,
            feed = feed,
            user = user,
        )
}
